app/helpers/course/discussion/topics_helper.rb

```diff
@@ -1,4 +1,12 @@
 # frozen_string_literal: true
 module Course::Discussion::TopicsHelper
   include Course::Discussion::CodeDisplayHelper
+
+  # Sanitize the title in settings.
+  #
+  # @return [String|nil] The formatted title.
+  def topics_title
+    # We don't want to return a blank string so this check is necessary.
+    @settings.title ? format_inline_text(@settings.title) : nil
```

Comment:
If you want to prevent a blank string, should it be `@settings.title.blank? ? nil : format_inline_text(@settings.title)` instead? 

Seems like I need to update group leaderboard title also >_<